JavaScript/Comments.tmPreferences

```diff
@@ -4,7 +4,7 @@
 	<key>name</key>
 	<string>Comments</string>
 	<key>scope</key>
-	<string>source.js, source.json</string>
+	<string>source.js</string>
```

Comment:
We might need to re-add comments to the JSON package even though JSON doesn't actually support them. See #285.

Reply:
There is a dedicated `Comments.tmPreferences` in the JSON package, just to avoid cross references. Hence we don't loose anything.